Automattic/wp-calypso
# WooCommerce Orders: Update the Add Order button

This PR removes the OrderHeader component- the single order & orders list view have different header actions, so it makes more sense to call ActionHeader directly. It also updated the Add Order link to go to wp-admin, since adding orders is not included in v1.

To test

1. View the orders list: `/store/orders/:site`
2. Clicking Add Order should take you to wp-admin
3. View a single order: `/store/order/:site/:orderId`
4. The action should now say Save Order, and clicking it does nothing (yet)
